XML/TestCases/FunctionalTests.xml

```diff
@@ -978,4 +978,17 @@
       <setupScript>.\Testscripts\Windows\SETUP-NET-Add-NIC.ps1</setupScript>
     </SetupConfig>
   </test>
+  <test>
+    <testName>GUI-GNOME-SHELL</testName>
+    <testScript>GUI-GNOME-SHELL.sh</testScript>
+    <files>.\TestScripts\Linux\GUI-GNOME-SHELL.sh,.\TestScripts\Linux\utils.sh</files>
+    <Platform>HyperV</Platform>
+    <Category>Functional</Category>
+    <Area>CORE</Area>
+    <Tags>gui</Tags>
```

Comment:
Try not to have many tags. Can you find one from the existing tags? Can you elaborate on why this is priority 2? Is it important?

Reply:
Thank you @juhlee-microsoft, change the tag to 'boot' since it is related to boot up when VM installs GUI.
Yes, it is important. If this case fails, VM may fail to login GUI. In my opinion, it should be priority 0 based on our testing. Considering you may not install GUI during testing, and this case will skip. It is low priority for you, then set as priority as 2. 
If you can agree to set priority as 0, it is great. We rely on priority 0 to run some basic tests locally. Thank you so much.